packages/os/pluto.service

```
@@ -0,0 +1,14 @@
+# pluto.service
+[Unit]
+Description=Generate additional settings for Kubernetes
+After=sundog.service
+Requires=sundog.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/pluto
```

These early services should commit settings first if they might depend on them being available:
```suggestion
ExecStartPre=/usr/bin/settings-committer
ExecStart=/usr/bin/pluto
```